staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -378,8 +379,8 @@ func (a *Admission) EvaluatePod(ctx context.Context, namespaceName string, podMe
 		klog.V(2).InfoS("failed to parse PodSecurity namespace labels", "err", err)
 		auditAnnotations["error"] = fmt.Sprintf("Failed to parse policy: %v", err)
 	}
-	// TODO: log nsPolicy evaluation with context (op, resource, namespace, name) for the request.
 
+	klog.V(2).InfoS("nsPolicy evaluation", "op", attrs.GetOperation(), "resource", attrs.GetResource(), "namespace", namespaceName, "name", attrs.GetName())
```

Comment:
Let's put this at a higher level, probably 4
```suggestion
	klog.V(4).InfoS("nsPolicy evaluation", "op", attrs.GetOperation(), "resource", attrs.GetResource(), "namespace", namespaceName, "name", attrs.GetName())
```

Reply:
```suggestion
	klog.V(2).InfoS("Policy evaluation", "op", attrs.GetOperation(), "resource", attrs.GetResource(), "namespace", namespaceName, "name", attrs.GetName())
```

Reply:
Lets log the policy too
```suggestion
	klog.V(2).InfoS("nsPolicy evaluation", "op", attrs.GetOperation(), "resource", attrs.GetResource(), "namespace", namespaceName, "name", attrs.GetName(), "policy", nsPolicy)
```